addons/sourcemod/scripting/shop/admin.sp

```diff
@@ -524,7 +519,13 @@ public int Admin_ItemsMenu_Handler(Menu menu, MenuAction action, int param1, int
 		case MenuAction_DrawItem  :
 		{
 			char info[16];
-			menu.GetItem(param2, info, sizeof(info));
+			int style;
+			menu.GetItem(param2, info, sizeof(info), style);
+			
+			if(!info[0])
```

Comment:
To prevent the passage of an empty line in the processing logic (the button is still not clickable).

Because in order to place the text "Empty category" you need to insert it as a MenuItem, otherwise there is no way (there must be at least one item in the menu - otherwise the menu will simply not appear and will be closed), I add this text, and since the menu logic has getting "identifier" - the first argument when adding `menu.AddItem()`, then I have an empty string there, the plugin gave an error .

Reply:
This error might be as result of wrong usage API, no?